pkgs/development/python-modules/pyclip/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, buildPythonPackage
+, pytest
+, xclip
+, xvfb-run
+}:
+
+buildPythonPackage rec {
+  pname = "pyclip";
+  version = "0.5.4";
+
+  src = fetchFromGitHub {
+    owner = "spyoungtech";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "19ff9cgnfx03mbmy5zpbdi986ppx38a5jf97vkqnic4g5sd1qyrn";
+  };
+
+  preConfigure = ''
+    sed -i 's@docs/README.md@README.md@g' setup.py
+  '';
+
+  checkInputs = [ pytest ] ++ lib.optionals stdenv.isLinux [ xclip xvfb-run ];
+
+  checkPhase = ''
+    ${lib.optionalString stdenv.isLinux "xvfb-run -s '-screen 0 800x600x24'"} pytest tests
+  '';
```

Comment:
Can we use pytestCheckHook here?

Reply:
The tests require a working X11 server to run xclip. The alternative would be to set-up Xvfb in preCheck and tear it down in postCheck. But using xvfb-run seems simpler.

Reply:
I think setting it up in preCheck might actually be nicer. We don't need to tear it down, that should be done automatically.

Anyway not that important right now.